echopype/tests/test_process_ad2cp.py

```diff
@@ -53,7 +53,12 @@ def test_process():
     "./echopype/test_data/ad2cp/SG198_0099_b.ad2cp.nc"
 ]
 
+THRESHOLD = 0.02
+
 def test_ocean_contour():
+    def close(a, b):
+        return abs(np.float32(a) - np.float32(b)) < THRESHOLD
```

Comment:
How did you decided `THRESHOLD = 0.02`? 

For some of the data variables below I can see the difference may be from rounding errors, but for things such as `base.attrs["Instrument_avg_nCells"]` and `base.attrs["Instrument_avg_nCells"]` why would the values not match exactly?

Also, why don't you use `np.isclose` for the comparison?